python/helpers/pydev/pydevd_file_utils.py

```diff
@@ -73,10 +73,29 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+def convert_to_long_pathname(filename):
```

Comment:
I have some concerns about this:
1) If we move the import ctypes inside the function it will cause a deadlock in the debugger. Since it will cause the importer lock to be acquired.  So if another thread is tracing a module level function then it will try to acquires  importer lock, then debugger lock. While our thread acquires debugger lock then importer lock. 

2) I'm trying to be mindful of performance since this function is executed once for every file traced and defining the function twice avoids performing the check every time for windows and ctypes.